Makefile

```diff
@@ -173,7 +176,7 @@ clean:
 
 .PHONY: distclean 
 distclean: clean
-	$(VE)$(MAKE) -C $(NXDK_DIR)/tools/extract-xiso clean $(QUIET)
+	$(VE)rm -rf $(NXDK_DIR)/tools/extract-xiso/build
```

Comment:
nit: Can we get away with just a `rm -r`? I'd rather not force `rm` if we don't have to.

Reply:
That would break `distclean` when `build` doesn't exist. I could offer `|| true` as an alternative though.

Reply:
👍 good point. I'd prefer `-f` then instead of silently allowing this line to fail.